lib/inputs/adzerk-impressions.js

```diff
@@ -25,10 +25,8 @@ module.exports = class AdzerkImpressions {
     } else {
       let hostCounts = {};
       return Promise.all(this._records.map(r => {
-        let host = url.parse(r.impressionUrl).host;
-        hostCounts[host] = hostCounts[host] || 0;
-        return pingurl.ping(r.impressionUrl, this._timeout, this._timeoutWait).then(
-          result => hostCounts[host]++,
+        return pingurl.ping(r.impressionUrl, r, this._timeout, this._timeoutWait).then(
```

Comment:
Note that this will start proxying agent/xff/referer to Adzerk as well.  I don't see any problem with that, though.

Reply:
yeah, seems like that could even be a good thing